### PR TITLE
Videos info xml

### PR DESCRIPTION
**Pull request by LifanPan**
Added a "New Video" notification
Remove Side bar
Added xml file containing video info
Added Javascript to generate video cards from xml

### CODE REVIEW

File: Videos/Smart Configurator Videos_Common.htm [hunk 1]
```diff
@@ -7,54 +7,80 @@
...
-                        <iframe src="https://players.brightcove.net/5260471205001/RyxO3haSE_default/index.html?videoId=6286881148001" 
-                        allowfullscreen=allowfullscreen></iframe>
-                        <h3>How to Create a Smart Configurator project in e² studio</h3>
-                        <div class="video-description">
-                            <p>This video demonstrates how to create a Smart Configurator project in e² studio.</p>
-                            <p>Date of upload: 2021-12-15</p>
-                        </div>
-                        <a href="https://www.renesas.com/video/how-create-smart-configurator-project-e-studio">View on Renesas website</a>
+                    <h1>Smart Configurator Videos</h1>
+                    <div class="info notification">
```
Comment (nam-ng) [inline Videos/Smart Configurator Videos_Common.htm:22]: Q: Do we want to show it if it is empty?

**Review by nam-ng — COMMENTED** [review]
Review-Time: 2m

**Review by leonardkong — COMMENTED** [review]
Looks good to me.

File: Videos/Smart Configurator Videos_Common.htm (hunk 1 above)
Comment (LifanPan) [reply Videos/Smart Configurator Videos_Common.htm:22]: Right now, this portion is written in HTML. It is possible to use javascript instead to only generate this portion if there are "new" videos. This way, this "New Videos" will not show up for Common Videos

**Review by leonardkong — APPROVED** [review]
Looks good to me.

**Pull request merged**